.github/workflows/ci.yaml

```diff
@@ -6,6 +6,10 @@ on:
     branches:
       - 'main'
 
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
```

Comment:
Just a suggestion
```suggestion
  group: ${{ github.head_ref || github.run_id }}-ci
```

Reply:
In the official reference, they actually use `${{ github.workflow }}` to specify the workflow.

Reply:
Thank you. I've fixed as you suggested.

Reply:
I mean, if we use `${{ github.workflow }}`, we don't need to add `-e2e` or `-ci`.